themes/helm/layouts/partials/sidebar.html

```diff
@@ -41,7 +41,7 @@
 
     <aside class="sidebar-buttons is-hidden-mobile">
       <div class="button-wrap">
-        <a href="https://github.com/helm/helm-www/tree/master/content/docs" class="button is-outline is-fullwidth">Contribute to Docs</a>
+        <a href="https://github.com/helm/helm-www/blob/master/CONTRIBUTING.md" class="button is-outline is-fullwidth">Contribute to Docs</a>
```

Comment:
The link might better be served as: https://github.com/helm/helm-www#contributing

Reply:
ok, I will update this link at once

Reply:
Completed

Reply:
sorry, I did not saw this issue [#530](https://github.com/helm/helm-www/issues/530) before submit this pr